spec/presenters/assessment_presenter_spec.rb

```diff
@@ -1,20 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe AssessmentPresenter do
-  
-  let(:claim)       { FactoryBot.create :claim, apply_vat: true }
-  
+  let(:claim) { FactoryBot.create :claim, apply_vat: true }
+  let(:presenter) { AssessmentPresenter.new(claim.assessment, view) }
```

Comment:
**[not a blocker/nit pick]** `AssessmentPresenter` is the class being tested so this could just be:

```ruby
subject(:presenter) { described_class.new(claim.assessment, view) }
```

Reply:
👍